### DISCUSSION TITLE
about moving sunlight

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#3846] [Q&A]
hiii i have a question

I can use the environment image in google model-editor, is there a way to completely remove it?

Actually 

I want to implement the movement of sunlight with time in google model editor, is it possible?

Or I want to change the position of the light according to time, is it possible in google model-viewer or model editor? 

please help me

**Comment by elalish** [comment]
We don't have an API for that currently. You could fork us or hack down into our three.js `$scene` using `getOwnSymbols` to accomplish it.

**Reply by ghost** [reply]
thank you. i will try it soon!
